### ISSUE TITLE
oauth login: should fail nicely instead of giving 500 when provider is down

### ISSUE THREAD

**Issue by david-caro** [inspirehep/inspire-next#2226]
Currently if there's any issue when trying to do the oauth login on inspire, the resulting page is a 500 error for the user, that is not helpful to the user, we should show a nicer message in the style "Failed to login, there's an unexpected issue with the oauth provider, please contact the admin at admin@inspirehep.net" or similar.

**Comment by jacquerie** [comment]
I agree with this, although, by looking at the stacktrace (https://sentry.cern.ch/inspire-sentry/inspire-labs/group/821670/), it looks like this has to be fixed in `invenio-oauthclient`, somewhere in this file: https://github.com/inveniosoftware/invenio-oauthclient/blob/7e8cd48f20a07ec97337650dda272a469c285c7a/invenio_oauthclient/views/client.py
